data/ms/legislature/Juan-Barnett-fcb35ef2-7463-4f8a-9a8a-3bc45b235ecb.yml

```diff
@@ -2,22 +2,24 @@ id: ocd-person/fcb35ef2-7463-4f8a-9a8a-3bc45b235ecb
 name: Juan Barnett
 given_name: Juan
 family_name: Barnett
-email: jtubb@house.ms.gov@senate.ms.gov
+email: jbarnett@senate.ms.gov
 image: http://billstatus.ls.state.ms.us/members/senate/barnett.jpg
 party:
 - end_date: '2021-12-14'
   name: Democratic
-- name: Republican
+- end_date: '2022-12-29'
```

Comment:
@NewAgeAirbender This person has always been a democrat as far as I can tell, so their party should be democrat and their end date should be January 2, 2024